src/Model/CachingIterator.php

```diff
@@ -87,7 +90,9 @@ public function count()
     #[ReturnTypeWillChange]
     public function current()
     {
-        return current($this->items);
+        /* Fetching current item */
```

Comment:
For consistency with other files in the project, single-line comments should use `//`. Alternatively, we can just remove these comments entirely as the subsequent line should be self-explanatory.